SDK_versioned_docs/version-3.0.0/guides/04-execute-a-swap.md

```diff
@@ -0,0 +1,249 @@
+---
+id: execute-a-swap
+title: Executing A Swap
+---
+
+This guide extends the previous [creating a pool](03-creating-a-pool.md) and [getting started](./using-ethers) guides by using the pool object to quote an estimated amount out for a trade, then creates a trade that can be submitted to the EVM.
+
+## Creating A Quoter Contract Object
+
+In order to retrieve a quote, we need to create a [Contract](https://docs.ethers.io/v5/api/contract/contract/) object using ethers.js
```

Comment:
can we change all instances of "we" and "our" to "you" and "your" for consistent second person voice? https://developers.google.com/style/person